cmd/ocagent/main.go

```diff
@@ -37,31 +37,30 @@ import (
 )
 
 func main() {
-	exportersYAMLConfigFile := flag.String("exporters-yaml", "config.yaml", "The YAML file with the configurations for the various exporters")
+	var configFile string
+	flag.StringVar(&configFile, "config", "config.yaml", "The YAML file with the configurations for the various exporters")
 	flag.Parse()
 
-	yamlBlob, err := ioutil.ReadFile(*exportersYAMLConfigFile)
+	yamlBlob, err := ioutil.ReadFile(configFile)
 	if err != nil {
-		log.Fatalf("Cannot read the YAML file %v error: %v", exportersYAMLConfigFile, err)
+		log.Fatalf("Cannot read the YAML file %v error: %v", configFile, err)
 	}
-	ownConfig, err := parseOCAgentConfig(yamlBlob)
+	agentConfig, err := parseOCAgentConfig(yamlBlob)
 	if err != nil {
-		log.Fatalf("Failed to parse own configuration %v error: %v", exportersYAMLConfigFile, err)
+		log.Fatalf("Failed to parse own configuration %v error: %v", configFile, err)
 	}
 
-	ocInterceptorAddr := ownConfig.openCensusInterceptorAddressOrDefault()
```

Comment:
We'll need to keep a mention of OpenCensus interceptor as we'll have more than the OpenCensus interceptor.